Makefile

```diff
@@ -178,6 +178,18 @@ deb-loader: mig-loader
 		--architecture $(FPMARCH) -v $(BUILDREV) \
 		-s dir -t deb .
 
+rpm-loader: mig-loader
+	rm -fr tmp
+	$(INSTALL) -s -D -m 0755 $(BINDIR)/mig-loader tmp/sbin/mig-loader
+	$(INSTALL) -D -m 0644 LICENSE tmp/usr/share/doc/mig-loader/copyright
```

Comment:
just a nitpick, why not keep LICENSE? (since its rather standard). r+ either way.